### DISCUSSION TITLE
System Requirements

### DISCUSSION THREAD

**Discussion by SatoX69** [coqui-ai/TTS#4074] [General Q&A] [answered]
What are the minimum system spec requirements to infer this and would it work on a mobile phone

**Comment by eginhard** [comment, marked as answer]
> What are the minimum system spec requirements to infer this

Depends on the model

> would it work on a mobile phone

See https://github.com/coqui-ai/TTS/discussions/3194

**Reply by SatoX69** [reply]
I checked, they all seem like basic speech engine, I was looking for something more robust, like elevenlab's tts which is multilingual natively
